gpu_ocean/SWESimulators/Common.py

```diff
@@ -821,7 +821,7 @@ class BoundaryConditions:
     """
     def __init__(self, \
                  north=1, east=1, south=1, west=1, \
-                 spongeCells=[0,0,0,0]):
+                 spongeCells={'north':0, 'south': 0, 'east': 0, 'west': 0}):
```

Comment:
Is this backward compatible?

I agree that it is easier to relate to 'east' than index 2, but if it breaks a lot of code I would say the timing is somewhat poor.

Reply:
Almost - I only found one place where this is being  used (simWrite to dump to netCDF), and I raise an appropriate exception there. Possibly in some notebooks, but that should be trivial to fix. 

I spent more time trying to figure out what the order of these four boundaries actually was than fixing all code that broke on my end.